spec/break_iterator_spec.cr

```diff
@@ -0,0 +1,15 @@
+require "./spec_helper"
+
+describe "ICU::BreakIterator" do
+  it do
+    ICU::BreakIterator.new("bla haha", LibICU::UBreakIteratorType::Word).to_a.should eq ["bla", " ", "haha"]
+  end
+
+  it do
+    ICU::BreakIterator.new("หน้าแรก", LibICU::UBreakIteratorType::Word).to_a.should eq ["หน้า", "แรก"]
```

Comment:
What alphabet is this ? The test seems to fail on old versions of ICU, I've checked the changelog, there is improvements in the break iterator in ICU [49](http://site.icu-project.org/download/49), [50](http://site.icu-project.org/download/50), [52](http://site.icu-project.org/download/52), ... This test may be changed to use an alphabet/sequence that will be handled the same way by all tested versions of ICU ?